internal/controller/drplacementcontrol_controller_test.go

```diff
@@ -701,7 +701,7 @@ func deleteDRPC() {
 	Expect(k8sClient.Delete(context.TODO(), drpc)).Should(Succeed())
 }
 
-func deleteNamespaceMWsFromAllClusters(namespace string) {
+func ensureNamespaceMWsDeletedFromAllClusters(namespace string) {
```

Comment:
This function should only check that there are no namespace MW left, and not delete them (line 712).